site/js/components/theme.js

```diff
@@ -29,6 +30,21 @@ export class Theme {
     this._name = value;
     this._elem.setAttribute('data-theme', value);
     localStorage.setItem('chromeos-theme', value);
+    this.dispatch();
+  }
+
+  /**
+   * Dispatch a custom event with the current name.
+   * @return {void}
+   */
+  dispatch() {
+    this._elem.dispatchEvent(
+      new CustomEvent('themeApplied', {
```

Comment:
Let's follow JS event naming conventions and make this `alllowercase` and name it as a change event

```suggestion
      new CustomEvent('themechange', {
```